src/openrct2-ui/drawing/engines/opengl/TextureCache.cpp

```diff
@@ -320,7 +321,9 @@ rct_drawpixelinfo TextureCache::GetGlyphAsDPI(uint32_t image, const PaletteMap&
     int32_t height = g1Element->height;
 
     rct_drawpixelinfo dpi = CreateDPI(width, height);
-    gfx_draw_sprite_palette_set_software(&dpi, ImageId::FromUInt32(image), -g1Element->x_offset, -g1Element->y_offset, palette);
+
+    const auto& glyphCoords = ScreenCoordsXY({ -g1Element->x_offset, -g1Element->y_offset });
```

Comment:
```suggestion
    const auto glyphCoords = ScreenCoordsXY{ -g1Element->x_offset, -g1Element->y_offset };
```
No need for the reference in here, as you've just allocated the value

Also, either use uniform initialization `{x , y}` or the constructor directly `(x ,y)`. In here theoretically you're creating a `ScreenCoordsXY` object and then calling the copy ctor. The compiler probably optimizes this away, but why add a layer of complexity when we can be simple to begin with :)